### ISSUE TITLE
How does this function?

### ISSUE THREAD

**Issue by scankannan** [iceixia/CRM_Auto_Reports#1]
Can you tell me if this is really in the works and if so what is the setup process?

**Comment by iceixia** [comment]
Yes this is the works, requested by a client. It's going to be a node.js application, cutting out sage and going for the database directly. The set up process when it's completed will be deployment to a service such as Heroku / Azure web apps or if required to a local instance of Node.js.
